### PR TITLE
docs: Document `isaric_raw` table a little more

### LINKED ISSUE #1375: Document `isaric` table
See https://github.com/opensafely-core/ehrql/issues/1340 for details.

### PR DESCRIPTION
**Pull request by StevenMaude**
Fixes #1375.

### CODE REVIEW

**Comment by StevenMaude** [comment]
Have asked researchers to give feedback here.

File: docs/includes/generated_docs/schemas/beta.raw.tpp.md [hunk 1]
```diff
@@ -172,14 +172,25 @@ This table contains some historical APCS data.
 <p class="dimension-indicator"><code>many rows per patient</code></p>
```
Comment (acagreen17) [inline docs/includes/generated_docs/schemas/beta.raw.tpp.md:172]: I think it's one row per patient?

File: docs/includes/generated_docs/schemas/beta.raw.tpp.md [hunk 2]
```diff
@@ -172,14 +172,25 @@ This table contains some historical APCS data.
...
 ## isaric
 
-A subset of the ISARIC data.
+A subset of the ISARIC data,
```
Comment (acagreen17) [inline docs/includes/generated_docs/schemas/beta.raw.tpp.md:175]: Can we expand on what a subset means?  I assume this relates to time frame and columns, but might be something else too?

File: docs/includes/generated_docs/schemas/beta.raw.tpp.md [hunk 3]
```diff
@@ -172,14 +172,25 @@ This table contains some historical APCS data.
...
+sourced from the [ISARIC COVID-19 Clinical Database][isaric_clinical_database].
 
 These columns are deliberately all taken as strings while in a preliminary phase.
```
Comment (acagreen17) [inline docs/includes/generated_docs/schemas/beta.raw.tpp.md:178]: All columns included have deliberately been taken as strings while in a preliminary phase.

File: docs/includes/generated_docs/schemas/beta.raw.tpp.md [hunk 4]
```diff
@@ -172,14 +172,25 @@ This table contains some historical APCS data.
...
 They will later change to more appropriate data types.
```
Comment (acagreen17) [inline docs/includes/generated_docs/schemas/beta.raw.tpp.md:179]: Will they actually?

File: docs/includes/generated_docs/schemas/beta.raw.tpp.md (hunk 1 above)
Comment (acagreen17) [reply docs/includes/generated_docs/schemas/beta.raw.tpp.md:172]: Oh must be something wrong with my code as I tried to count the number of admissions per person (using `dataset.n_admissions =  isaric_raw.count_for_patient()` and no one had more than one.

File: docs/includes/generated_docs/schemas/beta.raw.tpp.md (hunk 4 above)
Comment (StevenMaude) [reply docs/includes/generated_docs/schemas/beta.raw.tpp.md:179]: That's most likely if it gets promoted to a non-raw table, so we could remove this statement for now.

File: docs/includes/generated_docs/schemas/beta.raw.tpp.md (hunk 2 above)
Comment (StevenMaude) [reply docs/includes/generated_docs/schemas/beta.raw.tpp.md:175]: I've added a mention that this is referring to a subset of the columns in TPP. (There are something like ~900 columns in TPP.)

File: docs/includes/generated_docs/schemas/beta.raw.tpp.md (hunk 4 above)
Comment (StevenMaude) [reply docs/includes/generated_docs/schemas/beta.raw.tpp.md:179]: Done.

File: docs/includes/generated_docs/schemas/beta.raw.tpp.md (hunk 3 above)
Comment (StevenMaude) [reply docs/includes/generated_docs/schemas/beta.raw.tpp.md:178]: Edited to the suggestion here.

**Comment by StevenMaude** [comment]
Have updated this PR to include:

* @acagreen17's suggestions from this PR discussion thread
* @wjchulme's suggested introduction as mentioned on Slack

**Review by wjchulme — APPROVED** [review]

File: docs/includes/generated_docs/schemas/beta.raw.tpp.md (hunk 1 above)
Comment (wjchulme) [reply docs/includes/generated_docs/schemas/beta.raw.tpp.md:172]: I think conceptually it's still a many rows per patient table, so i think it's ok that it stays like this. Especially as, currently, there's greater query flexibility for these than for one row per patient tables

**Pull request merged**